app/src/main/java/org/mozilla/fenix/FeatureFlags.kt

```diff
@@ -103,11 +101,9 @@ object FeatureFlags {
      * Enables themed wallpapers feature.
      */
     fun isThemedWallpapersFeatureEnabled(context: Context): Boolean {
-        val strictMode = context.components.strictMode
-        return strictMode.resetAfter(StrictMode.allowThreadDiskReads()) {
```

Comment:
@MatthewTighe I had to remove strict mode ignore, as it was causing most of the tests to fail [see](https://github.com/mozilla-mobile/fenix/runs/5132271664?check_suite_focus=true), this is not visible in main as are not checking for `isThemedWallpapersFeatureEnabled` on [FenixApplication.kt](https://github.com/mozilla-mobile/fenix/pull/23655/files#diff-9e2b3999e8d0e181af6decd7d25949eef361031a719a578869967a5339b31737), I will continue further exploring tomorrow.

Reply:
Tested locally, this is a safe approach.